app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -206,6 +215,7 @@ export const ContentListView: React.FC = () => {
         <ContentToolBar onSearch={fetch} />
         <Row className="top-pane">
           <Row className="content-list">
+            <Loader visible={isLoading}></Loader>
```

Comment:
Move the loader functionality into the `FlexboxTable` so that we can take advantage of this everywhere we use the component.

Reply:
Updated, thanks